EPRN.Portal/Views/Waste/SubTypes.cshtml

```diff
@@ -31,7 +36,7 @@
             if (wasteSubType.AdjustmentPercentageRequired)
             {
                 <div class="govuk-radios__conditional" id="custom-percentage-div">
-                    <div class="govuk-hint">
+                    <p class="govuk-hint">
```

Comment:
According to the prototype, this should be a div, which is why i changed it to a div.